pkgs/development/python-modules/automatix/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyyaml
+, importlib-metadata
+, bundlewrap
+, bundlewrap-teamvault
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "automatix";
+  version = "1.12.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Lgft8lUUu8kKpoSutw7VxRz6Ktws2HG+0H7KfSoSFrA=";
+  };
+
+  # The dependency on bundlewrap and bundlewrap-teamvault is undeclared
+  propagatedBuildInputs = [ pyyaml importlib-metadata bundlewrap bundlewrap-teamvault ];
```

Comment:
If this is true, please report this issue upstream. As far as i understand the automatix documentation, the dependencies on `bundlewrap` and `bundlewrap-teamvault` should be loose, aka you only need those if you intend to use bundlewrap or teamvault in your automatix scripts.

Reply:
Fair point, those should probably be optional dependencies. I'll add a flag to make them optional.